contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -76,6 +72,22 @@ contract InitializableAbstractStrategy is IStrategy, Initializable, Governable {
         }
     }
 
+    /**
+     * @dev Single asset variant of the internal initialize.
+     */
+    function _initialize(
```

Comment:
This is also used by the CompoundStrategy. 
When we run migration 13 I don't think buidler will try to re-run the CompoundStrategy deployment but something I'm taking a mental note to watch for.

Reply:
It shouldn't do, I think you made the changes to the `.migration.json` which should be the source of truth for which migrations it runs. Changes to the underlying files shouldn't matter. It's important to be careful with these initializers though, we don't want to inadvertently allow reinitialization of an already initialized strategy.